calculator/parser.py

```diff
@@ -0,0 +1,137 @@
+BINARY_OPERATORS = ["+", "-", "*", "/"]
+
+
+def binary_operator(lhs, rhs, op):
+    """
+    Evaluates and acts on different binary operators
+
+    Parameters
+    ----------
+    lhs : integer or float
+        The first number to be operated on, will be on the left hand side of the op
+    
+    rhs : integer or float
+        The second number to be operated on, will be on the right hand side of the op
+    
+    op : string
+        The operator that is going to be applied to the numbers
+
+    Returns
+    -------
+    integer : The final answer after after the nums are operated on
+    """
+
+    if op == "+":
+        return lhs + rhs
+
+    elif op == "-":
+        return lhs - rhs
+
+    elif op == "*":
+        return lhs * rhs
+
+    elif op == "/":
+        return lhs / rhs
+    else:
+        raise ValueError("This operator is not supported, please try another one!")
```

Comment:
Can you give the user an indication of the operator that they used?  For instance `raise ValueError("{} is not a supported operator!".format(op))